shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -415,6 +421,11 @@ - (void)setMouseTrackingMode:(FlutterMouseTrackingMode)mode {
   [self configureTrackingArea];
 }
 
+- (void)setBackgroundColor:(NSColor*)color {
+  _backgroundColor = color;
```

Comment:
If you are going to cache it, you will also need to apply the background color during the loadView.